indexer/ftypes_matcher.hpp

```diff
@@ -173,6 +173,14 @@ class IsHotelChecker : public BaseChecker
   static vector<string> const & GetHotelTags();
 };
 
+class IsWifiChecker : public BaseChecker
```

Comment:
Please add a comment, that for history reasons WiFi is a type in classificator.txt, so it presence in a FeatureType should be checked with BaseChecker's derivative.